src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/TseHelper.java

```diff
@@ -93,11 +94,36 @@ public static DynamicFixedListType populateRespondentSelectApplication(CaseData
                 .toList());
     }
 
+    /**
+     * Create fields for application dropdown selector claimant rep.
+     *
+     * @param caseData contains all the case data
+     */
+    public static DynamicFixedListType populateClaimantRepSelectApplication(CaseData caseData) {
+        if (CollectionUtils.isEmpty(caseData.getGenericTseApplicationCollection())) {
+            return null;
+        }
+
+        return DynamicFixedListType.from(caseData.getGenericTseApplicationCollection().stream()
+                .filter(o -> !CLOSED_STATE.equals(o.getValue().getStatus())
```

Comment:
Correct me if I am wrong, does the closed state/status is still valid here given the change made as per Harpreet's comment on the configs?